.github/workflows/native-test-e2e-android.yml

```diff
@@ -0,0 +1,138 @@
+name: "[Test] suite-native Android E2E"
+on: [pull_request, workflow_dispatch]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build_android_test_app:
+    # run only if the PR contains `mobile-app` label or the action was dispatched manually
+    if: contains(github.event.pull_request.labels.*.name, 'mobile-app') || github.event_name == 'workflow_dispatch'
```

Comment:
Yes, that sounds good, together with `suite-common` I suppose? 

But also my question is, there could be changes in `yarn.lock` which could break the app by desktop team and this test could ideally prevent it. However I'm not sure whether it's wanted to run for all PR's. That might be more of a question for @matejkriz or @Nodonisko

Reply:
The test runs for ~20 minutes from what I see. However connect tests also take some time and run on every PR so I guess if it's not a required check for merging, we could run it on every PR.

Reply:
There is very low chance that changes in yarn.lock will break app so it's not worth it because it will trigger e2e in nearly every PR then.

Reply:
It will be much slower once we actually add test cases there, so maybe we should consider running it manually with e2e-mobile tag for example with combination when `suite-native/` and maybe `packages/connect` are modified.

Reply:
Agree with @Nodonisko manually with adding label mobile-app and checking for the changes in paths that are relevant for the mobile app.